test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -2101,6 +2110,24 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 					Expect(labels).To(HaveKeyWithValue("node.kubernetes.io/exclude-from-external-load-balancers", "true"))
 					Expect(labels).To(HaveKeyWithValue("node.kubernetes.io/exclude-disruption", "true"))
 				}
+				// Check node labels applied by cloud-node-manager
+				if hasAddon, _ := eng.HasAddon(common.CloudNodeManagerAddonName); hasAddon {
+					// Can't extract zone from API model, so just ensure that zone-related labels exist
+					Expect(labels).To(HaveKey("failure-domain.beta.kubernetes.io/zone"))
+					Expect(labels).To(HaveKey("topology.kubernetes.io/zone"))
+					region := eng.ExpandedDefinition.Location
+					Expect(labels).To(HaveKeyWithValue("failure-domain.beta.kubernetes.io/region", region))
+					Expect(labels).To(HaveKeyWithValue("topology.kubernetes.io/region", region))
+					var instanceType string
+					switch role {
```

Comment:
Where is the `role` variable defined?

Reply:
https://github.com/Azure/aks-engine/blob/98220f9895eebcfe674bd25019dc1c9f9c9dbbd4/test/e2e/kubernetes/kubernetes_test.go#L2097-L2104

Reply:
lolz thanks for clicking the accordion icon for me :)